docs/AppFramework.md

```diff
@@ -67,7 +67,7 @@ spec:
         secretRef: s3-secret
 ```
 
-6. Apply the Custom Resource specification: `kubectl apply -f Standalone.yaml`
+6. Apply the Custom Resource specification: `kubectl apply -n splunk-operator -f Standalone.yaml`
```

Comment:
Probably, specifying the name space as part of the CR itself may be a good option to avoid the confusion, as the `splunk-operator` namespace being kind of unique  here? Similarly for all other places